tests/test_fileclient.py

```diff
@@ -182,6 +182,37 @@ def test_lmdb_backend(self):
         img = mmcv.imfrombytes(img_bytes)
         assert img.shape == (120, 125, 3)
 
+    def test_http_backend(self):
+
+        def get_bytes(filepath):
+            with open(filepath, 'rb') as f:
+                content = f.read()
+            return content
+
+        http_backend = FileClient('http')
+
+        # input is path or Path object
+        with pytest.raises(Exception):
+            http_backend.get(self.img_path)
+        with pytest.raises(Exception):
+            http_backend.get(str(self.img_path))
+        with pytest.raises(Exception):
+            http_backend.get_text(self.text_path)
+        with pytest.raises(Exception):
+            http_backend.get_text(str(self.text_path))
+
+        # input url is http image
+        http_backend.get = MagicMock(return_value=get_bytes(self.img_path))
```

Comment:
We can use a real url to test this method, e.g., https://raw.githubusercontent.com/open-mmlab/mmcv/master/docs/mmcv-logo.png.

Reply:
It's now with real url links.